pkg/stream/server_frame.go

```diff
@@ -478,6 +482,29 @@ func (c *Client) metadataUpdateFrameHandler(buffer *bufio.Reader) {
 	}
 }
 
+func (c *Client) streamStatusFrameHandler(readProtocol *ReaderProtocol,
+	r *bufio.Reader) {
+
+	c.handleGenericResponse(readProtocol, r)
+
+	count, _ := readUInt(r)
+	streamStatus := make(map[string]int64)
+
+	for i := 0; i < int(count); i++ {
+		key := readString(r)
+		value := readInt64(r)
+		streamStatus[key] = value
+	}
+	res, err := c.coordinator.GetResponseById(readProtocol.CorrelationId)
+	if err != nil {
+		// TODO handle response
+		return
+	}
```

Comment:
If we are not returning the error, then should we log it?

Reply:
In this case, it can't find the correlation id.  added a log btw